headers/Data.h

```diff
@@ -17,5 +17,3 @@ class Data {
 	~Data();
 

```

Comment:
Please remove unnecessary lines.

Reply:
1. Triangle Class: Ensure that the Triangle class has its own destructor and memory management is done properly if it holds any dynamically allocated resources.
~Data() {
        // Properly delete dynamically allocated Triangle objects
        for (Triangle* t : trs) {
            delete t;
        }